migrations/Version202310191436392752_taoEventLog.php

```diff
@@ -20,19 +20,23 @@ public function getDescription(): string
     public function up(Schema $schema): void
     {
         if ($this->getExtensionsManager()->isEnabled('taoDacSimple')) {
-            $this->getEventManager()->attach(
+            $eventManager = $this->getEventManager();
+            $eventManager->attach(
                 'oat\taoDacSimple\model\event\DacChangedEvent',
                 [LoggerService::class, 'log']
             );
+            $this->getServiceLocator()->register(EventManager::SERVICE_ID, $eventManager);
```

Comment:
Thanks for this fix! How did you manage to find it? :) Are your local installation older than 2024?